Cargo.toml

```diff
@@ -42,3 +42,6 @@ bevy = { version = "0.11", default-features = false, features = [
     "bevy_pbr",
     "bevy_core_pipeline",
 ] }
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+bevy = { version = "0.11", features = ["webgl2"] }
```

Comment:
I guess the alternative is to target webgpu, but as it's mostly experimental, it should be fine ? Or we expose this control through a feature flag so user can choose 🤔

Reply:
solved by updating ci to use unstable web_sys api (hopefully)